Source/WebKit/NetworkProcess/NetworkSessionCreationParameters.cpp

```diff
@@ -100,7 +100,7 @@ void NetworkSessionCreationParameters::encode(IPC::Encoder& encoder) const
     encoder << isBlobRegistryTopOriginPartitioningEnabled;
 
     encoder << unifiedOriginStorageLevel;
-    encoder << perOriginStorageQuota << perThirdPartyOriginStorageQuota << originQuotaRatio << totalQuotaRatio << volumeCapacityOverride;
+    encoder << perOriginStorageQuota << originQuotaRatio << totalQuotaRatio << volumeCapacityOverride;
```

Comment:
Removal of perThirdPartyOriginStorageQuota is not really described in change log or in title.
Let's clarify this there.

Reply:
> How about rounding the volume capacity up to the GB and then keep the computation the way it is since the issue seems to be variance in volumeCapacity?

This might also work; I will make the change and update test.